### PR TITLE
Better tools

### PR DESCRIPTION
**Pull request by kristjanmik**
Added a custom opinionated seeder(will do for now).
Unified the yarn commands
Updated the README.md to reflect the changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Community Fund
 
-![](https://codeship.com/projects/a4ad0b40-eac5-0134-c6ec-52c39acfad49/status?branch=master)
```
Comment (johannth) [inline README.md:3]: Don't we want this?

File: README.md [hunk 2]
```diff
@@ -19,15 +17,25 @@ To run the project:
...
+
+Commands:
++ yarn start - Start the node server.
++ yarn start: all - Start the local db, seed the db and start the node server.
++ yarn db:start - Start the local db.
++ yarn db:stop - Stop the local db.
++ yarn db:seed - Seed the local db.
++ yarn db:flush - Flush the local db.
++ yarn lint:js - Lint all the JS files
++ yarn lint:css - Lint all the CSS files
```
Comment (johannth) [inline README.md]: Nit: maybe indent those slightly more so they get formatted as code.

File: README.md [hunk 3]
```diff
@@ -19,15 +17,25 @@ To run the project:
...
 
 + The CommunityFund website - http://localhost:3001
 + [Browsersync](https://browsersync.io/) - http://localhost:3002. This is an awesome debugging tool.
 + GraphQL interface - http://localhost:3001/graphql
++ A postgres database running within a docker container
 
 When running locally the database is run within a docker container. You can kill the local database (and flush all of the data) by running the following command:
 
-  yarn stop-dev-db
+  yarn db:stop
```
Comment (johannth) [inline README.md]: Nit: Maybe indent slightly more so it looks like code.

File: package.json [hunk 4]
```diff
@@ -218,6 +218,12 @@
     ]
   },
   "scripts": {
+    "start": "babel-node tools/run start",
+    "start:all": "yarn db:start && sleep 5 && yarn db:seed && yarn start",
```
Comment (johannth) [inline package.json]: We might want to add `yarn db:stop` first so that start:all always works. What do you think?

File: README.md [hunk 5]
```diff
@@ -19,15 +17,25 @@ To run the project:
         export DB_PASSWORD="communityfund_password"
 3. Run start
 
-        source .env && yarn start-dev-db
-        source .env && yarn start
+        yarn start:all
```
Comment (johannth) [inline README.md:22]: What do we want to do with `.env`? Do we want the users to source it before hand or should we bake it into the commands?

**Review by johannth — CHANGES_REQUESTED** [review]
Very nice changes. Need to decide what to do with `.env` because this fails without sourcing first.

File: README.md (hunk 1 above)
Comment (kristjanmik) [reply README.md:3]: This was a mistake!

File: README.md (hunk 5 above)
Comment (kristjanmik) [reply README.md:22]: We are already sourcing in tools/startDevDatabase.sh
Not sure about the way to go. Probably forcing it existence and baking it into the code is the right thing to do for now.

**Comment by kristjanmik** [comment]
@johannth thanks for the review, I will merge the changes for now, and we can open a separate PR for different issues.

**Pull request merged**